contrib/util/check-config.sh

```diff
@@ -55,6 +55,10 @@ is_set_as_module() {
 }
 
 color() {
+  if [[ -n "$NO_COLOR" ]]; then
```

Comment:
Nit: posix sh `[]` not bash `[[]]`
```suggestion
  if [ -n "$NO_COLOR" ]; then
```